IoTHubClients/CS/README.md

```diff
@@ -0,0 +1,83 @@
+# Azure IoT Hub Client Sample
+
+This sample demonstrates how to connect to IoT Hub, send telemetry, and monitor and respond to device twin state changes.
+
+The sample is Microsoft IoT Central compatible. See below for a complete walk-through.
+
+# Walk-through: Connecting to Microsoft IoT Central
+
+
+#### In this article
+
+This article describes how, as a device developer, to connect a device running a Windows 10 IoT Enterprise device (like Up-Board) to your Microsoft IoT Central application using the C# programming language.
+
+### Before you begin
+
+To complete the steps in this article, you need the following:
+
+- A Microsoft IoT Central application created from the Sample Devkits application template. For more information, see [Create your Microsoft IoT Central Application](https://docs.microsoft.com/en-us/microsoft-iot-central/howto-create-application).
+- A device running the Windows 10 IoT Enterprise system.  For this walkthrough, we will use a Up-Board.  
```

Comment:
"a Up-Board" should be "an Up-Board"

Reply:
Made the changes as mentioned in the readme document